backend/ethereum/bindings/abi.go

```diff
@@ -0,0 +1,52 @@
+// Copyright 2021 - See NOTICE file for copyright holders.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package bindings
+
+import (
+	"strings"
+
+	"github.com/ethereum/go-ethereum/accounts/abi"
+
+	"perun.network/go-perun/backend/ethereum/bindings/adjudicator"
+	"perun.network/go-perun/backend/ethereum/bindings/assetholder"
+	"perun.network/go-perun/backend/ethereum/bindings/assetholdererc20"
+	"perun.network/go-perun/backend/ethereum/bindings/assetholdereth"
+	"perun.network/go-perun/backend/ethereum/bindings/peruntoken"
+	"perun.network/go-perun/backend/ethereum/bindings/trivialapp"
+)
+
+// This file contains all the parsed ABI definitions of our contracts.
+// Use it together with `bind.NewBoundContract` to create a bound contract.
+
+var (
+	ERC20TokenABI, AdjudicatorABI, AssetHolderABI, ETHAssetHolderABI, ERC20AssetHolderABI, TrivialAppABI abi.ABI
```

Comment:
There is a linter warning: `exported var AdjudicatorABI should have its own declaration (golint)`